slurm/versions.tf

```diff
@@ -7,5 +7,5 @@ terraform {
 }
 
 provider "nebius" {
-  domain = "api.eu-north1.nebius.cloud:443"
+  domain = "api.eu.nebius.cloud:443"
```

Comment:
Hello! For soperator terraform we have specific release process, we cant merge you changes to main branch. We have branch `dev/soperator`, all changes should be merged to it, then we will create and bump version of release synchroniasly with our [main soperator code base](https://github.com/nebius/soperator)
So, please make separate PR as I described

Reply:
You can change the beach you merge to right in this PR on the GitHub UI

Reply:
I have reverted soperator reverted changes. Please, apply them yourself however you see fit.